src/Integration.UnitTests/Binding/ProjectBindingOperation.WriterTests.cs

```diff
@@ -1,27 +1,28 @@
 //-----------------------------------------------------------------------
-// <copyright file="ProjectRuleSetWriterTests.cs" company="SonarSource SA and Microsoft Corporation">
+// <copyright file="ProjectBindingOperation.WriterTests.cs" company="SonarSource SA and Microsoft Corporation">
```

Comment:
I merge the writers with the binding operations (as partial class to allow SxS comparison of the changes). The reason I did that is in order not to carry the ISourceControlledFileSystem all over the place when in fact the binding operation is about writing stuff and the previous split made the code more complex needlessly.